oop_Session1/A.java

```diff
@@ -0,0 +1,28 @@
+package oop_Session1;
+/**
+ *  Anchor is a HTML tag, in this we denote Anchor as "A"
+ *  Anchor is type of Atomic Element
+ * @author Akshita Dixit
+ *
+ */
+public class A extends AtomicElement {
+	
+	// Attribute of Anchor tag
+	String href;

```

Comment:
In general, class variables should be `private`, and exposed through "getter" methods.